File: Task1-Server/EventSuite.API/EventSuite.Core/DTOs/Requests/SmartBracelet/SmartBraceletRequest.cs
using EventSuite.Core.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EventSuite.Core.DTOs.Requests.SmartBracelet
{
    public class SmartBraceletRequest
    {
        public string? SerialNumber { get; set; }
        public SmartBraceletStatus Status { get; set; } = SmartBraceletStatus.Active;
        public DateTime? StartUsageDate { get; set; }
        public DateTime? EndUsageDate { get; set; }
        public double? AccessLatitude1 { get; set; }
        public double? AccessLatitude2 { get; set; }
        public double? AccessLongitude1 { get; set; }
        public double? AccessLongitude2 { get; set; }
        public double? CurrentLatitude { get; set; }
        public double? CurrentLongitude { get; set; }
    }
}

File: apzkr-pzpi-21-3-lukianchuk-danylo/Task3-WebClient/EventSuiteWeb/src/app/api/models/smart-bracelet-request.ts
/* tslint:disable */
/* eslint-disable */
import { SmartBraceletStatus } from '../models/smart-bracelet-status';
export interface SmartBraceletRequest {
  accessLatitude1?: number | null;
  accessLatitude2?: number | null;
  accessLongitude1?: number | null;
  accessLongitude2?: number | null;
  currentLatitude?: number | null;
  currentLongitude?: number | null;
  endUsageDate?: Date;
  serialNumber?: string | null;
  startUsageDate?: Date;
  status?: SmartBraceletStatus;
}
